README.md

```diff
@@ -46,10 +46,12 @@ OpenTelemetry is the merging of OpenCensus and OpenTracing into a single project
 This project contains the following top level components:
 
 * [OpenTelemetry API](api/):
-  * [stable apis](api/all/src/main/java/io/opentelemetry/api/) including `Tracer`, `Span`, `SpanContext`, `Meter`, and `Baggage`
-  * [context api](context/src/main/java/io/opentelemetry/context/) The OpenTelemetry Context implementation.
-* [extensions](extensions/) define additional API extensions, which are not part of the core API.
+  * [stable apis](api/all) including `Tracer`, `Span`, `SpanContext`, `Meter`, and `Baggage`.
+  * [context api](context/) The OpenTelemetry Context implementation.
+  * [experimental apis](api/all) including `Events`.
```

Comment:
(not sure what you intended here)
```suggestion
  * [experimental apis](api/events) including `Events`.
```

Reply:
Yup that's what I meant.. Because enumerating everything _except_ events seems like some sort of intention omission. This will make more sense when we resolve #6121.